src/app/store/action/auth.action.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { Action } from '@ngrx/store';
+import { ILoginCredentials, IUser } from 'src/app/model/user.model';
+
+export enum EAuthActions {
```

Comment:
that `E` in front of the enum name is usually not done, rename it to `AuthActionTypes` for example